ci/docker/Dockerfile

```diff
@@ -15,10 +15,18 @@ COPY ci/rust.version /var/rust.version
 # "ant python git": build sample Buck project.
 RUN apt-get update --quiet > /dev/null && \
     apt-get --assume-yes --quiet install sudo openssh-server openjdk-8-jdk \
-    golang clang build-essential lib32stdc++6 lib32z1 unzip ant python git curl && \
-    export MAINFRAMER_RUST_VERSION="$(cat /var/rust.version)" && \
-    curl --silent --fail --location https://static.rust-lang.org/rustup.sh | sh -s -- --revision="$MAINFRAMER_RUST_VERSION" && \
-    rustc --version && cargo --version
+    golang clang build-essential lib32stdc++6 lib32z1 unzip ant python git curl
+
+RUN groupadd build_user && \
```

Comment:
This step used to happen in entrypoint, but new Rust toolchain doesn't support multiuser installation, I've tried to mess with exposing it for build_user but couldn't make it work.

Thus I add `build_user` at image build time and install Rust toolchain to that user.

Entrypoint just has to make sure that files are owned by proper uid/gid